stratum/hal/lib/nikss/nikss_switch.cc

```diff
@@ -0,0 +1,117 @@
+#include "stratum/hal/lib/nikss/nikss_switch.h"
+
+#include "absl/memory/memory.h"
+#include "absl/synchronization/mutex.h"
+
+extern "C" {
+#include "nikss/nikss.h"
+}
+
+namespace stratum {
+namespace hal {
+namespace nikss {
+
+NikssSwitch::NikssSwitch(PhalInterface* phal_interface,
+                         NikssChassisManager* nikss_chassis_manager)
+    : phal_interface_(ABSL_DIE_IF_NULL(phal_interface)),
+      nikss_chassis_manager_(ABSL_DIE_IF_NULL(nikss_chassis_manager)) {}
+
+NikssSwitch::~NikssSwitch() {}
+
+::util::Status NikssSwitch::PushChassisConfig(const ChassisConfig& config) {
+  return ::util::OkStatus();
```

Comment:
Hmm, some sort of error maybe should be returned to mark that method is not implemented yet? I mean every function in this file